src/helpers/config.ts

```diff
@@ -79,21 +78,37 @@ export async function extractChainId(conn: Config = {}): Promise<number> {
   return chainId;
 }
 
-const findOrCreateFile = function (filepath: string): Buffer {
+const findOrCreateFile = async function (filepath: string): Promise<Buffer> {
+  const fs = await getFsModule();
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
   if (!fs.existsSync(filepath)) {
     fs.writeFileSync(filepath, JSON.stringify({}));
   }
 
   return fs.readFileSync(filepath);
 };
 
+// TODO: next major we should remove the jsonFileAliases helper and expose it
+//    in a different package since it doesn't work in the browser.
+const getFsModule = (function () {
+  let fs: any;
+  return async function () {
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+    if (fs) return fs;
+
+    fs = await import(/* webpackIgnore: true */ "fs");
```

Comment:
In addition to not importing "fs" unless the dev is using the helper, I also have to tell webpack to ignore this to get the build to work.